dev-tools/mage/config.go

```diff
@@ -205,6 +219,14 @@ func subheader(title string) string {
 	return makeHeading(title, "-")
 }
 
+var nonWhitespaceRegex = regexp.MustCompile(`(?m)(^.*\S+.*$)`)
```

Comment:
```suggestion
var nonWhitespaceRegex = regexp.MustCompile(`(?m)(^.*\S.*$)`)
```

Any non-whitespace chars should cause indentation (not more than 1).